x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/capacity/AutoscalingDeciderContext.java

```diff
@@ -24,4 +26,8 @@
      * Return the nodes governed by the policy.
      */
     Set<DiscoveryNode> nodes();
+
+    ClusterInfo info();
+
+    SnapshotShardSizeInfo snapshotShardSizeInfo();
```

Comment:
Could you add Javadocs to these new methods, and also `state`?

Reply:
:+1:, 22456be13356107f7d98e09b40a593d2a8de3220